admin/data/sql/migrations/20191011004727.do.create-service-message-table.sql

```diff
@@ -0,0 +1,29 @@
+CREATE TABLE [mtc_admin].[serviceMessage] (
+  id int IDENTITY (1,1) NOT NULL,
+  createdAt datetimeoffset(3) NOT NULL DEFAULT GETUTCDATE(),
+  updatedAt datetimeoffset(3) NOT NULL DEFAULT GETUTCDATE(),
+  version rowversion,
+  recordedByUser_id int NOT NULL,
+  updatedByUser_id int NULL,
+  title nvarchar(max) NOT NULL,
+  message nvarchar(max) NOT NULL,
+  isDeleted bit NOT NULL DEFAULT 0,
```

Comment:
Is this soft-delete flag required? Why?  I would rather not have it unless it is absolutely required.

Reply:
@jon-shipley changed to hard delete.